chain-abci/src/app/mod.rs

```diff
@@ -1,3 +1,6 @@
+#[macro_use]
```

Comment:
is this still needed in Rust 2018: https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html ? It's not needed on crates, not sure about modules

Reply:
https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html#more-details
Yes, it's still needed for locally defined macros.